### PR TITLE
UIOR-230 default instance status setting

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
Add setting for specifying instance-status to use when creating instances in mod-orders
https://issues.folio.org/browse/UIOR-230

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/common/constants/api.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-// eslint-disable-next-line import/prefer-default-export
 export const LINES_API = 'orders/order-lines';
+export const INSTANCE_STATUSES_API = 'instance-statuses';
```
Comment (AndreiShumski2) [inline src/common/constants/api.js]: Now we have two files with API paths: this one and `src/components/Utils/api.js` .
Should we keep it in one place?

File: src/common/constants/constants.js [hunk 2]
```diff
@@ -22,3 +22,5 @@ export const ORDERS_DOMAIN = 'orders';
 export const NOTE_TYPES = {
   poLine: 'poLine',
 };
+
+export const CONFIG_INSTANCE_STATUS = 'inventory-instanceStatusCode';
```
Comment (AndreiShumski2) [inline src/common/constants/constants.js]: the same. Is it better to push it into `src/components/Utils/const.js`?

File: src/common/constants/api.js (hunk 1 above)
Comment (aliaksei-chumakou) [reply src/common/constants/api.js]: we keep common code under`common` dir as a pattern in all acquisitions repos. `src/components/Utils/api.js` - is a legacy and a deprecated way. I don't see too much sense to hustle around keeping all in one place for now

File: src/common/constants/constants.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/common/constants/constants.js]: the same

**Pull request merged**